From 80acdb7f8b027db8953b7067d29ee2283bde42bb Mon Sep 17 00:00:00 2001
From: Christian Hewitt <christianshewitt@gmail.com>
Date: Tue, 3 Aug 2021 18:42:55 +0000
Subject: [PATCH 2/7] LOCAL: board: amlogic: odroid-n2: remove /amlogic/ prefix
 for dtb path

Remove the /amlogic/ prefix to align with current LE dtb locations.

Signed-off-by: Christian Hewitt <christianshewitt@gmail.com>
---
 board/amlogic/odroid-n2/odroid-n2.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/board/amlogic/odroid-n2/odroid-n2.c b/board/amlogic/odroid-n2/odroid-n2.c
index 2135457edd9..ec1f4efc113 100644
--- a/board/amlogic/odroid-n2/odroid-n2.c
+++ b/board/amlogic/odroid-n2/odroid-n2.c
@@ -63,7 +63,7 @@ static void odroid_set_fdtfile(char *soc, char *variant)
 {
 	char s[128];
 
-	snprintf(s, sizeof(s), "amlogic/meson-%s-odroid-%s.dtb", soc, variant);
+	snprintf(s, sizeof(s), "meson-%s-odroid-%s.dtb", soc, variant);
 	env_set("fdtfile", s);
 }
 
-- 
2.34.1

